tests/v2/validation/projects/container_default_resource_limit_validation_test.go

```diff
@@ -0,0 +1,308 @@
+	pattern := fmt.Sprintf(`admission webhook "rancher.cattle.io.projects.management.cattle.io" denied the request: project.spec.containerDefaultResourceLimit: Invalid value: v3.ContainerResourceLimit{RequestsCPU:"%s", RequestsMemory:"%s", LimitsCPU:"%s", LimitsMemory:"%s"}: requested memory %s is greater than limit %s`, cpuReservation, memoryReservation, cpuLimit, memoryLimit, memoryReservation, memoryLimit)
+	require.Regexp(pcrlv.T(), regexp.MustCompile(pattern), err.Error())
+}
+
+func (pcrlv *ProjectsContainerResourceLimitValidationTestSuite) TestValidCpuLimitButMemoryLimitLessThanRequest() {
+	subSession := pcrlv.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a project in the downstream cluster with CPU limit greater than the CPU Request but Memory limit set lower than the Memory Request. Verify that the webhook rejects the request.")
+	cpuLimit := "200m"
+	cpuReservation := "100m"
+	memoryLimit := "32Mi"
+	memoryReservation := "64Mi"
+	_, err := createProject(pcrlv.standardUserClient, pcrlv.cluster.ID, "", "", cpuLimit, cpuReservation, memoryLimit, memoryReservation)
+	require.Error(pcrlv.T(), err)
+	pattern := fmt.Sprintf(`admission webhook "rancher.cattle.io.projects.management.cattle.io" denied the request: project.spec.containerDefaultResourceLimit: Invalid value: v3.ContainerResourceLimit{RequestsCPU:"%s", RequestsMemory:"%s", LimitsCPU:"%s", LimitsMemory:"%s"}: requested memory %s is greater than limit %s`, cpuReservation, memoryReservation, cpuLimit, memoryLimit, memoryReservation, memoryLimit)
+	require.Regexp(pcrlv.T(), regexp.MustCompile(pattern), err.Error())
+}
+
+func (pcrlv *ProjectsContainerResourceLimitValidationTestSuite) TestCpuAndMemoryLimitEqualToRequest() {
+	subSession := pcrlv.session.NewSession()
+	defer subSession.Cleanup()
+
+	log.Info("Create a project (with CPU and Memory limit equal to Request) and a namespace in the project.")
+	cpuLimit := "200m"
+	cpuReservation := "200m"
+	memoryLimit := "64Mi"
+	memoryReservation := "64Mi"
+	createdProject, createdNamespace, err := createProjectAndNamespace(pcrlv.standardUserClient, pcrlv.cluster.ID, "", "", cpuLimit, cpuReservation, memoryLimit, memoryReservation)
+	require.NoError(pcrlv.T(), err)
+
+	log.Info("Verify that the container default resource limit in the Project spec is accurate.")
+	projectSpec := createdProject.Spec.ContainerDefaultResourceLimit
+	require.Equal(pcrlv.T(), cpuLimit, projectSpec.LimitsCPU, "CPU limit mismatch")
+	require.Equal(pcrlv.T(), cpuReservation, projectSpec.RequestsCPU, "CPU reservation mismatch")
+	require.Equal(pcrlv.T(), memoryLimit, projectSpec.LimitsMemory, "Memory limit mismatch")
+	require.Equal(pcrlv.T(), memoryReservation, projectSpec.RequestsMemory, "Memory reservation mismatch")
+
+	log.Info("Verify that the namespace has the label and annotation referencing the project.")
+	updatedNamespace, err := namespacesApi.GetNamespaceByName(pcrlv.standardUserClient, pcrlv.cluster.ID, createdNamespace.Name)
```

Comment:
This is necessary because the label field.cattle.io/projectId may not be immediately available in the namespace when it is created within the project.

Reply:
thanks for letting me know!